guides/source/active_job_basics.md

```diff
@@ -122,7 +122,11 @@ GuestsCleanupJob.set(wait: 1.week).perform_later(guest)
 GuestsCleanupJob.perform_later(guest1, guest2, filter: 'some_filter')
 ```
 
-That's it!
+NOTE: Jobs are meant to be run with the `perform_later` and
+`perform_now` wrappers: running the job's `perform` method directly is
+possible but removes the benefit of using a queue and skips all the
+logic provided by the ActiveJob wrappers like callbacks or exception
```

Comment:
As `ActiveJob` in this case references the component, not the Ruby module, it should have a space.
See also: https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#wording

```suggestion
logic provided by the Active Job wrappers like callbacks or exception
```